app/next-client-app/app/(public)/layout.tsx

```diff
@@ -10,8 +11,11 @@ export default async function PublicLayout({
   const user = await getCurrentUser();
```

Comment:
I'd rather take out this request for now - as most hits to the landing page will then result in a 401 to the backend.

Reply:
I agree. To know whether the user logged in or not (to render the menu bar items accordingly), I now need to use cookies. Hope it will be okay